docs/UserGuide.md

```diff
@@ -277,26 +281,78 @@ Format: `unprioritze INDEX`
 
 ### Marking assignments as done : `done`
 
-Marks the specified assignment as done.
+Format: `done INDEX [MORE_INDEXES]`
 
-Format: `done INDEX`
+You can keep track of your undone assignments by marking assignments that you've completed as done.
```

Comment:
Maybe phrasing it as `You can keep track of your uncompleted assignments...` would be better since `undone` isn't really grammatically correct in this scenario XD

Reply:
sounds good!